lib/srv6.h

```diff
@@ -90,6 +90,7 @@ struct srv6_locator {
 	uint64_t current;
 	bool status_up;
 	struct list *chunks;
+	uint8_t perchunk_bits_length;
```

Comment:
chunk_bits_length is better naming I feel, ref: `{block_bits,node_bits,function_bits,argument_bits}_length`